assets/js/race.js

```diff
@@ -208,8 +208,10 @@ function cacheIndividual(order) {
     var ifteam = $("[name='ifteam']").prop('checked');
     var shimano16 = $("[name='shimano16']").val();
     var shimano17 = $("[name='shimano17']").val();
-    data[order - 1] = {
-        order: order,
+    if (ifteam && (race == '0')) {
```

Comment:
So this is for some kind of hacking? Actually I don't like it.
We can restrict options by js in the feature.